presto-docs/src/main/sphinx/migration/from-hive.rst

```diff
@@ -132,3 +132,15 @@ Presto query::
 
     date_diff('day', startdate, enddate)
 
+Overwriting data on insert
+--------------------------
+
+By default, ``INSERT`` queries are not allowed to overwrite existing data. You
+can use the catalog session property ``insert_existing_partitions_behavior`` to
+allow overwrites. Prepend the name of the catalog using the Hive connector,
+e.g.g ``hdfs``, and set the property in the session before you run the insert
```

Comment:
Extra `.g`

Reply:
Why is `hdfs` used as an example name instead of `hive`?

Reply:
imho it is misleading to use the name of the connector for the catalog name .. so I avoid that confusion whenever possible ..

Reply:
Agreed. But hdfs is also a little misleading as it's too tied to the underlying technology and may make people think this is for hdfs only.